### DISCUSSION TITLE
`pytest` Fails on Windows

### DISCUSSION THREAD

**Discussion by adam-grant-hendry** [python-poetry/poetry#8410] [General]
When following the [Contributing Guide](https://python-poetry.org/docs/contributing/#code-contributions) for `Local development`, running `poetry run pytest` results in 199 errors. Is Windows supported with `poetry`?

![image](https://github.com/python-poetry/poetry/assets/59346180/cb4c07ce-4905-4f1a-a9a9-d9c09be33872)

![image](https://github.com/python-poetry/poetry/assets/59346180/8f35c5fd-bfb3-4ece-8674-345dfda3cb9a)

**Comment by dimbleby** [comment]
works fine in CI https://github.com/python-poetry/poetry/actions/runs/6057889829/job/16439495337

**Reply by adam-grant-hendry** [reply]
@dimbleby I believe this is a small false-positive because your [`isolate_environ` fixture](https://github.com/python-poetry/poetry/blob/02448cf7f184dea204156f7dcb620a4f01a0068e/tests/conftest.py#L272) overlooks the [environment variables `ANSICON`, `ConEmuAnsi`, and `TERM`](https://github.com/python-poetry/cleo/blob/e03ce6ffe6b8d94faf4999dc092362881a3619a9/src/cleo/io/outputs/stream_output.py#L91), which `cleo` can use
